test/migrators/division/never_division.hrx

```diff
@@ -8,16 +8,18 @@
   g: (six / three);
   h: 6 / three;
   i: #{$x} / #{6px / 3px} / #{2};
+  j: rgb(10 20 30 / 0.5);
 }
 
 <==> output/entrypoint.scss
-:foo(#{slash-list(6, 3)}) {
-  b: slash-list(6px, 3px);
-  c: slash-list(6px, 3px, 2);
+:foo(#{6/3}) {
+  b: 6px / 3px;
```

Comment:
Now that these are being preserved as `/`, it's probably a good idea to add additional test cases where they're assigned to variables so we can verify that they're converted to `slash-list()` when necessary.